specs/requests/artifacts_spec.py

```diff
@@ -102,6 +112,32 @@
                 with it("returns an empty list for tags"):
                     expect(self.response.json.get("tags")).to(equal([]))
 
+                with it("saves the image to the configured uploads directory"):
+                    file_name = self.response.json.get("url").split("/")[-1]
+                    file_path = os.path.join(current_app.config["UPLOAD_FOLDER"], file_name)
+                    expect(os.path.isfile(file_path)).to(equal(True))
+
+                with description("resized image versions"):
+                    with before.each:
+                        self.widths = [320, 480, 640, 750, 1080]
+
+                        original_file_name = self.response.json.get("url").split("/")[-1]
+                        [self.prefix, self.suffix] = original_file_name.rsplit('.', 1)
```

Comment:
Again the original file_name should not be calculated from the response.

Reply:
Same here. The original filename is the name Elija assigned to the file.